docs/usage-examples/find.txt

```diff
@@ -0,0 +1,107 @@
+.. _laravel-find-usage:
+
+=======================
+Find Multiple Documents
+=======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: find many, retrieve, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can retrieve multiple documents from a collection by calling the ``where()`` method
+on an Eloquent model or a query builder.
+
+Pass a query filter to the ``where()`` method to retrieve documents that meet a
+set of criteria. MongoDB returns the matching documents according to their
+:term:`natural order` in the database or according to the sort order that you can specify
+by using the ``orderBy()`` method.
+
+Example
+-------
+
+This usage example performs the following actions:
+
+- Uses the ``Movie`` Eloquent model to represent the ``movies`` collection in the
+  ``sample_mflix`` database
+- Retrieves documents from the ``movies`` collection that match a query filter
+
+The example calls the following methods on the ``Movie`` model:
+
+- ``where()``: matches documents in which the value of the ``runtime`` field is greater than ``800``
+- ``orderBy()``: sorts matched documents by their ascending ``_id`` values
+- ``get()``: retrieves the query results
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: ../includes/usage-examples/FindManyTest.php
+      :start-after: begin-find
+      :end-before: end-find
+      :language: php
+      :dedent:
+
+   .. output::
+      :language: console
+      :visible: false
+
+      // Results are truncated
+      
+      {
+      "_id": "...",
+      "plot": "The economic and cultural growth of Colorado spanning two centuries from the mid-1700s to the late-1970s.",
+      "genres": [
+          "Action",
+          "Adventure",
+          "Drama"
+      ],
+      "runtime": 1256,
+      ...
+      "title": "Centennial",
+      ...
```

Comment:
Suggestion:
I think showing the expanded "title", "runtime", and "_id" fields could make the output easier to read and would be sufficient to help the reader understand the query filter and sort order. I think the rest could be hidden with "..."

Perhaps you'd want matches that also aren't easy to mistake for 
another more popular title such as "Taken" since the "runtime" field could be a point of confusion.

Another solution could be to avoid using the "_id" sort and instead using the "runtime" value so that you could omit the "_id" field.